src/commands/edit-input-schema.js

```diff
@@ -9,19 +9,32 @@ const { cryptoRandomObjectId } = require('@apify/utilities');
 
 const { ApifyCommand } = require('../lib/apify_command');
 const outputs = require('../lib/outputs');
+const { readInputSchema, DEFAULT_INPUT_SCHEMA_PATHS } = require('../lib/input_schema');
 
 const INPUT_SCHEMA_EDITOR_BASE_URL = 'https://apify.github.io/input-schema-editor-react/';
 const INPUT_SCHEMA_EDITOR_ORIGIN = new URL(INPUT_SCHEMA_EDITOR_BASE_URL).origin;
 
-const DEFAULT_INPUT_SCHEMA_PATH = './INPUT_SCHEMA.json';
-
 // Not really checked right now, but it might come useful if we ever need to do some breaking changes
 const API_VERSION = 'v1';
 
 class EditInputSchemaCommand extends ApifyCommand {
     async run() {
         const { args } = this.parse(EditInputSchemaCommand);
-        const { path = DEFAULT_INPUT_SCHEMA_PATH } = args;
+        let path = null;
+
+        // This call fails if the input schema is invalid JSON
+        const inputSchemaWithPath = await readInputSchema(args.path);
+
+        if (!inputSchemaWithPath) {
+            outputs.warning('Input schema has not been found. Default one will be created.');
+            fs.writeFileSync(DEFAULT_INPUT_SCHEMA_PATHS[0], '{}');
```

Comment:
If you specify a path manually (with `args.path`), the input schema should be created at that path, not at the default one.

Reply:
Or if it's specified in `actor.json : input`, then it should be created at that path.